draft-ietf-tls-subcerts.md

```diff
@@ -602,14 +534,14 @@ without requiring CAs to issue new intermediate certificates.
 
 # Operational Considerations
 
-The operational consideration documented in this section should be
+The operational considerations documented in this section should be
 taken into consideration when using Delegated Certificates.
```

Comment:
"Delegated Certificates" here should be "Delegated Credentials".

Reply:
```suggestion
taken into consideration when using Delegated Credentials.
```